src/es/components/pages/ClassicsSearch.html

```diff
@@ -31,6 +31,9 @@
             </a-button>
           </form>
       </m-form>
+      <div class="searchResultsContainer">
```

Comment:
its okay that you handle it outside the web component, but dont you think it would be easier if you just expect the result container within the component, then you can control its style... or maybe the result will once be its own component then you would load the result container (wc) through load child components... hide the form and show the result.

Reply:
additionally, if you make or handle the result container within the m-form, you can create it, in case it is not supplied.

Reply:
makes sense.. will update 👍🏼